pom.xml

```diff
@@ -1048,6 +1049,23 @@
                     <artifactId>sisu-maven-plugin</artifactId>
                     <version>${version.sisu-maven-plugin}</version>
                 </plugin>
+                <plugin>
+                    <groupId>io.github.dmlloyd.module-info</groupId>
+                    <artifactId>module-info</artifactId>
+                    <version>${version.module-info}</version>
+                    <executions>
+                        <execution>
+                            <id>module-info</id>
+                            <phase>prepare-package</phase>
+                            <goals>
+                                <goal>generate</goal>
+                            </goals>
+                            <configuration>
+                                <exportExcludes>^.*\b(impl|internal|_private|private_)\b.*$</exportExcludes>
```

Comment:
This is a new property. The plugin as it is now would not exclude `impl` packages, so I've added this regexp to make it configurable. An alternative would be to disable the automatic listing of exports and add them manually.